Classes/Private/Wrapper/OCTToxAV.m

```diff
@@ -617,7 +618,7 @@ void receiveAudioFrameCallback(ToxAV *cToxAV,
     OCTToxAV *toxAV = (__bridge OCTToxAV *)userData;
 
     dispatch_async(dispatch_get_main_queue(), ^{
-        DDLogCInfo(@"%@: receiveAudioFrameCallback from friend %d sampleCount: %lu channels: %d", toxAV, friendNumber, sampleCount, channels);
+        // DDLogCInfo(@"%@: receiveAudioFrameCallback from friend %d sampleCount: %lu channels: %d samplerate: %d", toxAV, friendNumber, sampleCount, channels, sampleRate);
```

Comment:
oops shouldn't be :smile: Removed it temporarily for testing to see if it had any effect.